testutil/daemon/daemon.go

```diff
@@ -334,9 +335,9 @@ func (d *Daemon) StartWithLogFile(out *os.File, providedArgs ...string) error {
 		dockerdBinary = "sudo"
 		d.args = append(d.args,
 			"-u", d.rootlessUser.Username,
-			"-E", "XDG_RUNTIME_DIR="+d.rootlessXDGRuntimeDir,
-			"-E", "HOME="+d.rootlessUser.HomeDir,
-			"-E", "PATH="+os.Getenv("PATH"),
+			"--preserve-env",
+			"XDG_RUNTIME_DIR="+d.rootlessXDGRuntimeDir,
+			"HOME="+d.rootlessUser.HomeDir,
```

Comment:
Note for reviewers: the `sudo -E` flag takes no arguments. `sudo -E HOME=/home/rootlessuser` is parsed as two separate flags: preserve all environment variables, and set the `HOME` variable. The extra `-E` flags were redundant. The only functional change I have made is to stop overriding the `PATH` variable.

Reply:
What was the reason for removing PATH from the list? Depending on the environment, PATH may be a protected env-var, and therefore reset to its default (which can lead to fun situations; see https://github.com/moby/term/pull/21#issuecomment-720093769)

Reply:
I was hoping to pass through `PATH` from the `os/exec.Cmd.Args` so the path override in my integration test (`daemon.WithEnvVars`) would Just Work in rootless mode. Guess I'll need to find a different way to accomplish that.

Reply:
I guess that would still be possible 🤔 as in; when it would run `sudo -E PATH=/foo:/bar PATH=/bar/baz`, the second `PATH` would override the former (default "inherited") path, correct?